tex/chProgramming.tex

```diff
@@ -2186,22 +2186,19 @@ \section{Notations, abbreviations}\label{sec:notabrev}
 \item Postfix notations begin with \C{.}, as in \C{.+1} and \C{.-group}
 	to let one write \C{(p.-group G)}. There is one exception for
         the postfix notation of the factorial, which starts with
-        \C{`}, as in \C{m`}\C{!}.
-        % [DG] Workaround using two \C environments, since a single
-        % \C{m`!} compiles to an "m" followed by an inverted
-        % exclamation mark.
+        \C{`}, as in \D{m`!}.
```

Comment:
Shouldn't ``\C{`}`` be replaced with ``\D{`}`` as well? ``\C{`}`` results in a plain backtick, but I think it should be typeset in typewriter face (sorry for nitpicking).